server/data/vocabularies.json

```diff
@@ -0,0 +1,2246 @@
+                "name": "Turkey",
+                "is_active": true
+            },
+            {
+                "qcode": "TKM",
+                "name": "Turkmenistan",
+                "is_active": true
+            },
+            {
+                "qcode": "TCA",
+                "name": "Turks and Caicos Islands",
+                "is_active": true
+            },
+            {
+                "qcode": "TUV",
+                "name": "Tuvalu",
+                "is_active": true
+            },
+            {
+                "qcode": "UGA",
+                "name": "Uganda",
+                "is_active": true
+            },
+            {
+                "qcode": "UKR",
+                "name": "Ukraine",
+                "is_active": true
+            },
+            {
+                "qcode": "ARE",
+                "name": "United Arab Emirates",
+                "is_active": true
+            },
+            {
+                "qcode": "GBR",
+                "name": "United Kingdom",
+                "is_active": true
+            },
+            {
+                "qcode": "USA",
+                "name": "United States of America",
+                "is_active": true
+            },
+            {
+                "qcode": "UMI",
+                "name": "US Minor Outlying Islands",
+                "is_active": true
+            },
+            {
+                "qcode": "URY",
+                "name": "Uruguay",
+                "is_active": true
+            },
+            {
+                "qcode": "UZB",
+                "name": "Uzbekistan",
+                "is_active": true
+            },
+            {
+                "qcode": "VUT",
+                "name": "Vanuatu",
+                "is_active": true
+            },
+            {
+                "qcode": "VEN",
+                "name": "Venezuela (Bolivarian Republic)",
+                "is_active": true
+            },
+            {
+                "qcode": "VNM",
+                "name": "Viet Nam",
+                "is_active": true
+            },
+            {
+                "qcode": "VIR",
+                "name": "Virgin Islands, US",
+                "is_active": true
+            },
+            {
+                "qcode": "WLF",
+                "name": "Wallis and Futuna Islands",
+                "is_active": true
+            },
+            {
+                "qcode": "ESH",
+                "name": "Western Sahara",
+                "is_active": true
+            },
+            {
+                "qcode": "YEM",
+                "name": "Yemen",
+                "is_active": true
+            },
+            {
+                "qcode": "ZMB",
+                "name": "Zambia",
+                "is_active": true
+            },
+            {
+                "qcode": "ZWE",
+                "name": "Zimbabwe",
+                "is_active": true
+            }
+        ]
+    },
+    {
+        "_id": "asset_formats",
```

Comment:
I think that we don't need `asset_formats`. 
Please, clarify it with @karelpetrak

Reply:
same as above

Reply:
Yes, agree with @ride90. The list of asset formats reflects the way the data are stored in Belga's database now. Users will not interact with it in SD. Thus we can leave them out.